script/files/parallel_runtime_rspec.log

```diff
@@ -0,0 +1,443 @@
+spec/controllers/work_packages/bulk_controller_spec.rb:99.308038
```

Comment:
I will improve this at a later point to re-write it with every run and download it from somewhere. I have it on my list.